expr/functions/multiplySeriesWithWildcards/function.go

```diff
@@ -69,10 +69,24 @@ func (f *multiplySeriesWithWildcards) Do(ctx context.Context, e parser.Expr, fro
 	}
 
 	results := make([]*types.MetricData, 0, len(nodeList))
+	commonTags := helper.GetCommonTags(args)
+	commonTags["aggregatedBy"] = "multiply"
 
 	for _, series := range nodeList {
 		args := groups[series]
-		r := args[0].CopyTag("multiplySeriesWithWildcards("+series+")", map[string]string{"name": series})
+		r := args[0].CopyLink()
+		r.Name = series
+		r.Tags = make(map[string]string)
```

Comment:
May be encapsulate logic for tags copy in additional Copy function ? Many cases in this PR need them for simplify logic.